pom.xml

```diff
@@ -44,19 +44,43 @@
             </roles>
             <timezone>+4</timezone>
         </developer>
+        <developer>

```

Comment:
Hehe, misunderstood that. I'm happy being listed among the "Contributors" in the README.md.
*removed*